tree/tree/src/InternalTreeUtils.cxx

```diff
@@ -66,11 +66,12 @@ namespace TreeUtils {
 /// \param[in] treeName Name of the tree.
 /// \param[in] fileNameGlob Path to the file. Refer to TChain::Add for globbing rules.
 /// \param[in] alias Alias for this friend.
-void RFriendInfo::AddFriend(const std::string &treeName, const std::string &fileNameGlob, const std::string &alias)
+void RFriendInfo::AddFriend(const std::string &treeName, const std::string &fileNameGlob, const std::string &alias, Long64_t treeEntries)
 {
    fFriendNames.emplace_back(std::make_pair(treeName, alias));
    fFriendFileNames.emplace_back(std::vector<std::string>{fileNameGlob});
    fFriendChainSubNames.emplace_back();
+   fFriendEntries.emplace_back(std::vector<Long64_t>{treeEntries});
```

Comment:
if you construct the `std::vector` temporary anyway, you might as well use `push_back` :smile: (I know, like 72 has the same issue)

Reply:
These modifications to the member functions `AddFriend` are not strictly necessary for this PR anyway. They were there to provide some symmetry with the new addition to the constructor of RFriendInfo (i.e. being able to retrieve the number of entries of each friend after construction and one by one). But for now it's not necessary so in the next commit I remove these